docs/pages/project/contributing/contributing-model-relationships.md

```diff
@@ -0,0 +1,293 @@
+---
+layout: page
+title: Contributing to Model Relationships
+permalink: project/contributing/contributing-model-relationships
+redirect_from: project/contributing/contributing-model-relationships/
+abstract: How to contribute to Meshery Models Relationships, Policies...
+language: en
+type: project
+category: contributing
+list: include
+---
+
+Relationships within Model play a crucial role in establishing concrete visualisations of efficient data flow between different components of Meshery. These are used to classify the nature of interaction between one or more interconnected Components.
+
+1. Identify the relationship and any specific constraints to be enforced between the two specific components, their models or potentially other components, models, or environmental considerations.
+2. Propose a specific visual representation for the relationship.
+3. Visual representation examples:
+
+   - [Hierarchical]({{ site.baseurl }}/assets/img/meshmodel/relationships/hierarchical_relationship.png)
+   - [Sibling]({{ site.baseurl }}/assets/img/meshmodel/relationships/sibling_relationship.png)
+   - [Binding]({{ site.baseurl }}/assets/img/meshmodel/relationships/binding_relationship.png)
+   - [Edge]({{ site.baseurl }}/assets/img/meshmodel/relationships/mount_edge_relationship.png)
+
+4. Prospose the appropriate relationship type, using one of the predefined set of relationship types or suggest a new relationship where an existing type does not fit.
+5. Create a Relationship Definition (yaml).
+6. Create a policy for evaluation of the relationship (rego). See [examples](https://github.com/meshery/meshery/tree/master/server/meshmodel/policies/).
+7. Add in Documentation.
+
+### Existing Relationships, their Definitions and their Subtypes
+
+1. `Hierarchical` relationships involve either an ancestral connection of the components i.e. the creation/ deletion of a Component higher up affects the existence of the Components below in the lineage or a connection which involves the inheritence of features from one Component to the other.
+
+- [Parent](https://github.com/meshery/meshery/tree/master/server/meshmodel/kubernetes/relationships/) - A parent-child relationship implies the requirement of the parent component before the child component can be created. For example, a "Namespace" can be a parent of "Pods" within that namespace. The namespace must exist before creating pods within it.
```

Comment:
@pandeyshubham03, we have to use fewer links redirecting to the Meshery GitHub since continuous changes in the repository may lead to the links being deprecated. Right @leecalcote ?

Reply:
Yes. Moreover this link is specific to one model, but is being presented to the reader as if it represents all models. If this link is offered, we'll want to also indicate that it leads to an example / source code. That isn't evident based on the word "Parent".